xml/deployment_installation.xml

```diff
@@ -825,14 +825,350 @@
+         <link xlink:href="https://docs.openstack.org/newton/user-guide/common/cli-install-openstack-command-line-clients.html">
+         install a local <command>openstack</command> client</link>.
+        </para>
+        <para>
+         To use the local client, you need to access <guimenu>Project &rarr; Compute &rarr; 
+         Access &amp; Security</guimenu> in the Horizon Dashboard and click on the 
+         <guimenu>Download &ostack; RC File v3</guimenu>.
+        </para>
+        <para>
+         The downloaded file is a script that you then need to load using the 
+         <command>source</command> command. The script will ask you for your 
+         &soc; password.
+        </para>
+         <screen>
+&prompt.user;<command>source container-openrc.sh</command>
+         </screen>
+       </note>
+       <procedure>
+
+         <step>
+          <para>
+           Upload the container image to &ostack; Glance (Image service). This
+           example uses the name <literal>CaaSP-3</literal> as the name of the 
+           image that is created in &soc;.
+          </para>
+         <screen>
+&prompt.user;<command>openstack image create --public --disk-format qcow2 \
+--container-format bare \
+--file SUSE-CaaS-Platform-3.0-OpenStack-Cloud.x86_64-1.0.0-GM.qcow2 \
+<replaceable>CaaSP-3</replaceable></command>
+         </screen>
+         </step>
+         <step>
+          <warning>
+           <title>Replace the default <replaceable>root_password</replaceable></title>
+            <para>
+             Do not use the <filename>caasp-environment.yaml</filename> directly
+             from the GitHub repository.
+            </para>
+             <para>
+              You must make sure to replace the value for 
+              <literal>root_password</literal> with a secure password.
+              This will become the password for the &rootuser; account on all 
+              nodes in the stack.
+             </para>
+           <screen>
+---
+parameters:
+  root_password: <replaceable>linux</replaceable>
+  admin_flavor: m1.large
+&hellip;
+           </screen>
+         </warning>
+          <para>
+           Download the <filename>caasp-stack.yaml</filename> and 
+           <filename>caasp-environment.yaml</filename> Heat templates to your 
+           workstation and then run the <command>openstack stack create</command>
+           command.
+          </para>
+          <screen>
+&prompt.user;<command>openstack stack create \ 
+-t caasp-stack.yaml \
+-e caasp-environment.yaml \
+--parameter image=<replaceable>CaaSP-3</replaceable> <replaceable>caasp3-stack</replaceable></command>
+          </screen>
+         </step>
+
+        <step>
+         <para>
+          Find out which (external) IP address was assigned to the admin node
+          of your &productname; cluster (here: <literal>10.81.1.51</literal>).
+         </para>
+          <screen>
+&prompt.user;<command>openstack server list --name "admin"</command>
```

Comment:
This screen is way to wide. Either run it again on a 80*25 terminal (or 100x25 if it doesn't fit) or manually shorten it here.

Reply:
What would "manually shorten" mean? I already removed all unnecessary whitespace from the output that didn't cull any information

Reply:
The output on a narrower terminal
```
| 7969fa8c-6b55-419b-9586-90c612b5c95a | caasp3-stack-admin                     | ACTIVE | caasp3-stack-caasp-int-net=172.24.0.15, 10.81.1.51       | CaaSP-3                           | m1.large  |
```